resources/js/components/Navbar.vue

```diff
@@ -80,20 +81,8 @@ const mobileMenu = ref(false)
                 </Link>
             </template>
         </div>
-        <Transition>
-            <div v-if="mobileMenu" class="flex absolute left-0 h-[150vh] w-full bg-[rgba(0,0,0,.3)]">
-                <div class="w-[80%] bg-white z-10 h-full">
-                    <div>
-                        <div v-if="user">
-                            hello
-                            <img :src="user.avatar_url" :alt="user.name">
-                        </div>
-                        <div v-else>
-
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </Transition>
+        <transition>
+            <MobileMenu v-if="mobileMenu" />
```

Comment:
Truyền props `user` qua cho `MobileMenu` component
```suggestion
            <MobileMenu v-if="mobileMenu" :user="user" />
```